packages/core/test/insured-bridge/BridgePool.js

```diff
@@ -1059,8 +1042,8 @@ describe("BridgePool", () => {
         toBN(await l1Token.methods.balanceOf(relayer).call())
           .sub(toBN(relayerBalanceBefore))
           .toString(),
-        realizedSlowRelayFeeAmount.toString(),
-        "Slow relayer should receive proposal slow relay reward"
+        toBN(totalRelayBond).add(realizedSlowRelayFeeAmount).toString(),
```

Comment:
This was a false positive test that passed before because it equates `relayerBalanceBefore - relayerBalanceNow` and `realizedSlowRelayFeeAmount`, and `relayerBalanceBefore` was unintentionally being called AFTER  the `optimisticOracle.settle()` method, so the relayer had already received back their bond by that time.

Reply:
great catch. we should spend some time working on these tests and making sure they are doing what we intend. there are a number of other simplifications and syntactical changes I'm keen to make once we've got the other stuff 100% locked down.